activesupport/lib/active_support/tagged_logging.rb

```diff
@@ -71,11 +69,7 @@ def current_tags
 
       def tags_text
```

Comment:
Where is this method called now?

Reply:
What if this method now receives a message with default `""` we use it in the `call` passing `msg`?

Reply:
> Where is this method called now?

In `ActionDispatch::DebugExceptions#log_array`:

https://github.com/rails/rails/blob/13783f36ad6c4adbe2121272bd0c4d239bb153ab/actionpack/lib/action_dispatch/middleware/debug_exceptions.rb#L153-L154

> What if this method now receives a message with default `""` we use it in the `call` passing `msg`?

In the most recent commit, I am now doing something similar to that by calling [`tag_stack.format_message("")`](https://github.com/rails/rails/pull/46955/files#diff-68f90ba4998610be2f7da027e006e8ddeeb06dece66f41579c94070e12f23011R65). :+1: